cylc/flow/config_schema.py

```diff
@@ -0,0 +1,660 @@
+                'submission failed handler': [VDR.V_STRING_LIST, None],
+                'warning handler': [VDR.V_STRING_LIST, None],
+                'critical handler': [VDR.V_STRING_LIST, None],
+                'retry handler': [VDR.V_STRING_LIST, None],
+                'submission retry handler': [VDR.V_STRING_LIST, None],
+                'execution timeout handler': [VDR.V_STRING_LIST, None],
+                'submission timeout handler': [VDR.V_STRING_LIST, None],
+                'custom handler': [VDR.V_STRING_LIST, None],
+            },
+            'suite state polling': {
+                'user': [VDR.V_STRING],
+                'host': [VDR.V_STRING],
+                'interval': [VDR.V_INTERVAL],
+                'max-polls': [VDR.V_INTEGER],
+                'message': [VDR.V_STRING],
+                'run-dir': [VDR.V_STRING],
+                'verbose mode': [VDR.V_BOOLEAN],
+            },
+            'environment': {
+                '__MANY__': [VDR.V_STRING],
+            },
+            'directives': {
+                '__MANY__': [VDR.V_STRING],
+            },
+            'outputs': {
+                '__MANY__': [VDR.V_STRING],
+            },
+            'parameter environment templates': {
+                '__MANY__': [VDR.V_STRING],
+            },
+        },
+    },
+    'suite logging': {
+        'rolling archive length': [VDR.V_INTEGER, 5],
+        'maximum size in bytes': [VDR.V_INTEGER, 1000000],
+    },
+    'suite run platforms': {
+        'run hosts': [VDR.V_SPACELESS_STRING_LIST],
+        'run ports': [VDR.V_INTEGER_LIST, list(range(43001, 43101))],
+        'condemned hosts': [VDR.V_ABSOLUTE_HOST_LIST],
+        'auto restart delay': [VDR.V_INTERVAL],
+        'run host select': {
+            'rank': [VDR.V_STRING, 'random', 'load:1', 'load:5', 'load:15',
+                     'memory', 'disk-space'],
+            'thresholds': [VDR.V_STRING],
+        },
+        'suite host self-identification': {
+            'method': [VDR.V_STRING, 'name', 'address', 'hardwired'],
+            'target': [VDR.V_STRING, 'google.com'],
+            'host': [VDR.V_STRING],
+        },
+    },
+    'visualization': {
+        'initial cycle point': [VDR.V_CYCLE_POINT],
+        'final cycle point': [VDR.V_STRING],
+        'number of cycle points': [VDR.V_INTEGER, 3],
+        'collapsed families': [VDR.V_STRING_LIST],
+        'use node color for edges': [VDR.V_BOOLEAN],
+        'use node fillcolor for edges': [VDR.V_BOOLEAN],
+        'use node color for labels': [VDR.V_BOOLEAN],
+        'node penwidth': [VDR.V_INTEGER, 2],
+        'edge penwidth': [VDR.V_INTEGER, 2],
+        'default node attributes': [
+            VDR.V_STRING_LIST, ['style=unfilled', 'shape=ellipse']],
+        'default edge attributes': [VDR.V_STRING_LIST],
+        'node groups': {
+            '__MANY__': [VDR.V_STRING_LIST],
+        },
+        'node attributes': {
+            '__MANY__': [VDR.V_STRING_LIST],
+        },
+    },
```

Comment:
I am sure you will need to revisit this when we have a clearer idea the graph view and/or `cylc graph` replacement.

Reply:
Yes the current visualization section is quite closely aligned to graphviz styling conventions.

Reply:
@wxtim - I think you can remove the entire visualization section for now.  We can add back in whatever we need, once we have the new graph tech in place.  Probably need to obsolete the vis section of cylc-7 suites, as auto-upgrade doesn't seem feasible (at least not at this point).

Reply:
Fair. Will do.